### PR TITLE
Hide layout field for tabs/wizard/accordion in edit dialog

### PR DESCRIPTION
**Pull request by dmaurya929**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by rismehta** [comment]
Please squash and merge to a single commit since commits are anyways not correctly organized. This also helps in generating appropriate release notes

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/_cq_dialog/.content.xml [hunk 1]
```diff
@@ -34,9 +34,77 @@
                 maximized="{Boolean}true">
```
Comment (rismehta) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/_cq_dialog/.content.xml:34]: Shouldn't the base resource type of wizard in the inheritance be base ?

Comment (dmaurya929) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/_cq_dialog/.content.xml:34]: Do you mean `resourceType` mentioned above  ? 
In the base cq:dialog that is `sling:resourceSuperType="core/wcm/components/panelcontainer/v1/panelcontainer"` ,  resourceType of tabs is  same so it doesn't have any affect, should I remove it line 33 then ?

**Comment by dmaurya929** [comment]
@rismehta can we merge this PR ?

**Pull request merged**